packages/flutter_tools/lib/src/cmake.dart

```diff
@@ -25,19 +27,66 @@ String _escapeBackslashes(String s) {
   return s.replaceAll(r'\', r'\\');
 }
 
+class _Version {
+  const _Version(this.major, this.minor, this.patch);
+
+  factory _Version.fromBuildName(String buildName) {
+    if (buildName == '1.0.0') {
```

Comment:
Why is 1.0 special-cased?

Reply:
We should not add complexity to code for micro-optimizations that we don't have evidence makes a meaningful difference; we especially shouldn't do so when we already know they won't be.

Reply:
Removed